locales/zh_TW/pcsx2_Iconized.po

```diff
@@ -3,18 +3,17 @@ msgstr ""
 "Project-Id-Version: PCSX2 1.3.1 1201\n"
```

Comment:
Version can be bumped to 1.6.0

Reply:
Got it. Will be changed in the next commit.